<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicClass</class>
 <widget class="QMainWindow" name="GraphicClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow 
{
	border-image: url(./Img/wallpaper-14461.jpg);
}

QStatusBar
{
	background-color: rgba(156, 156, 156, 200);
	border-width: 2px;
	border-radius: 8px;
	border-color: rgb(0, 0, 0);
	min-width: 10em;
	padding: 2px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget_13">
       <layout class="QGridLayout" name="gridLayout_37">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_38">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Dimitri Swank</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(156, 156, 156, 200);
border-width: 2px;
border-radius: 8px;
border-color: rgb(0, 0, 0);
padding: 2px;</string>
                </property>
                <property name="text">
                 <string>User</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_9">
                <property name="enabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_28">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="lineEdit">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>MS Reference Sans Serif</family>
                  <pointsize>10</pointsize>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(156, 156, 156, 200);
border-width: 2px;
border-radius: 8px;
border-color: rgb(0, 0, 0);
padding: 2px;</string>
                </property>
                <property name="text">
                 <string>Personnal message...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="4">
         <widget class="Line" name="line_27">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_41">
          <item row="0" column="0" colspan="5">
           <widget class="Line" name="line_26">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_27">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>35</width>
              <height>25</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_13">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Call</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_14">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="deleteSelectedFriendPushButton_7">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer_26">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>36</width>
              <height>25</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" colspan="5">
           <widget class="Line" name="line_25">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_39">
          <item row="0" column="0">
           <widget class="QTabWidget" name="FriendTabWidget_7">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="Friends_7">
             <attribute name="title">
              <string>Friends</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_40">
              <item row="0" column="0" colspan="2">
               <widget class="FriendListWidget" name="listWidget_7">
                <property name="styleSheet">
                 <string notr="true">background-image: url(./Img/wallpaper-14461.jpg);
</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="Nothing_7">
             <attribute name="title">
              <string>Nothing</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_14">
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <layout class="QGridLayout" name="gridLayout_42">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_23">
              <item>
               <layout class="QGridLayout" name="gridLayout_43">
                <item row="0" column="0">
                 <widget class="QLabel" name="selectedFriendNameLabel_8">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>Dimitri Swank</family>
                    <pointsize>14</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(156, 156, 156, 200);
border-width: 2px;
border-radius: 8px;
border-color: rgb(0, 0, 0);
min-width: 10em;
padding: 2px;</string>
                  </property>
                  <property name="text">
                   <string>Selected Friend</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="selectedFriendIconStatusLabel_8">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Icon</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <spacer name="horizontalSpacer_29">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>34</width>
                    <height>25</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="selectedFriendPersonalMsgLabel_8">
                  <property name="font">
                   <font>
                    <family>MS Reference Sans Serif</family>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgba(156, 156, 156, 200);
border-width: 2px;
border-radius: 8px;
border-color: rgb(0, 0, 0);
min-width: 10em;
padding: 2px;</string>
                  </property>
                  <property name="text">
                   <string>Selected Friend Personal Message</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <spacer name="horizontalSpacer_30">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>34</width>
                    <height>25</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="activeFriendCallPushButton_8">
                  <property name="text">
                   <string>Call</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_29">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTextBrowser" name="friendMsgBox_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>2</horstretch>
                <verstretch>3</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>800</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255, 200);
border-width: 2px;
border-radius: 8px;
border-color: rgb(0, 0, 0);
min-width: 10em;
padding: 2px;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="sendBoxTextEdit_8">
                <property name="maximumSize">
                 <size>
                  <width>700</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(255, 255, 255, 230);
border-width: 2px;
border-radius: 8px;
border-color: rgb(0, 0, 0);
min-width: 10em;
padding: 2px;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sendBoxPushButton_8">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVBabel">
    <property name="title">
     <string>VBabel</string>
    </property>
    <addaction name="actionCall"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuManage_account">
    <property name="title">
     <string>Manage account</string>
    </property>
    <addaction name="actionCreateAccount"/>
    <addaction name="actionDelete_account"/>
   </widget>
   <widget class="QMenu" name="menuVoicemail">
    <property name="title">
     <string>Voicemail</string>
    </property>
    <addaction name="actionSet_voicemail"/>
    <addaction name="actionUnset_voicemail"/>
   </widget>
   <widget class="QMenu" name="menuFriends_list">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Friends list</string>
    </property>
    <addaction name="actionAdd_friend"/>
    <addaction name="actionDelete_friend"/>
   </widget>
   <addaction name="menuVBabel"/>
   <addaction name="menuConnection"/>
   <addaction name="menuManage_account"/>
   <addaction name="menuFriends_list"/>
   <addaction name="menuVoicemail"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="font">
    <font>
     <family>MS Reference Sans Serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionCreateAccount">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create account</string>
   </property>
   <property name="font">
    <font>
     <family>MS Reference Sans Serif</family>
    </font>
   </property>
  </action>
  <action name="actionLogin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSet_voicemail">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set voicemail</string>
   </property>
  </action>
  <action name="actionUnset_voicemail">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Unset voicemail</string>
   </property>
  </action>
  <action name="actionCall">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Call</string>
   </property>
   <property name="font">
    <font>
     <family>MS Reference Sans Serif</family>
    </font>
   </property>
  </action>
  <action name="actionDelete_account">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete account</string>
   </property>
  </action>
  <action name="actionAdd_friend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add friend</string>
   </property>
  </action>
  <action name="actionDelete_friend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete friend</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FriendListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">friendlistwidget.h</header>
   <slots>
    <slot>slot1(QListWidgetItem*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sendBoxTextEdit_8</sender>
   <signal>textEdited(QString)</signal>
   <receiver>friendMsgBox_8</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
